src/Umbraco.Web/UmbracoHelper.cs

```diff
@@ -289,6 +289,56 @@ public IPublishedContent Member(string id)
             return asInt ? MembershipHelper.GetById(asInt.Result) : MembershipHelper.GetByProviderKey(id);
         }
 
+        public IEnumerable<IPublishedContent> Member(IEnumerable<int> ids)
```

Comment:
Sorry to keep this PR lying around so long but keep seeing things we might not want. I know you mentioned this last round too but the non plural on this seems odd. I know it seems consistent with multiple results for `Content` and `Media`, but both of those terms are interestingly both plural and singular (like Moose) whereas Member is singular and Members is plural. So what are your thoughts? It seems odd to have multiple members returned by `Member` and these should maybe be `Members` ? I realize it doesn't seem consistent but we can't have `Contents` or `Medias` since that doesn't actually make sense.

Reply:
It's perfectly fine 👍 rather have it correct the first time than having to live with it to be backwards compatible.

I don't mind renaming the member methods to plural on the helper. I see exactly the same pros and cons as you do. Would people be surprised by the plural notation when they're used to work with content and media? Maybe. Maybe not. Is it inconsistent to use plural? Yes. Does it make sense anyway? Yes. 

At the end of the day I don't think the consistency matters a whole lot to the devs in this matter. The "don't make me think" argument wins for me. And I'm normally really big on consistency.

Reply:
Fine by me, we'll go with the "don't make me think" argument 😂 , i should get this merged in tomorrow.

Reply:
Excellent! I'll update to plural notation for members all 'round. There's also consistency in having it be `Members` on both helpers... so my need for consistency is still satisfied. Win-win!

Reply:
Et voila - 6d9ed6c